woocommerce/woocommerce
# Feature/quantity placeholder

### All Submissions:


### Changes proposed in this Pull Request:

Allow the `woocommerce_quantity_input()` function and the `quantity-input.php` template to support a placeholder attribute.

Closes # .

### How to test the changes in this Pull Request:

Use the following snippet:

```
function kia_add_placeholder_attr_to_inputs( $args, $product ) {
	$args['placeholder'] = '0';
	$args['input_value'] = '';
	return $args;
}
add_filter( 'woocommerce_quantity_input_args', 'kia_add_placeholder_attr_to_inputs', 10, 2  );
```

A product should now an empty quantity input with a 0 placeholder. This is not that practical, but it shows it works. A place where this would be beneficial in core would be in grouped products. I can add a commit for that if you are open to it.

In Mix and Match I've had customers complain about needing to remove the 0 before being able to type in a value and I agree that's not good UX.

https://share.getcloudapp.com/qGuD7mkm

### Other information:




### Changelog entry

> Add support for placeholder attribute in quantity inputs.
